resources/locales.clj

```diff
@@ -16,13 +19,16 @@
    "zh-HK"
    "fr"
    "pt-BR"
+   "zh-CN"
    "de"
    "nb"
    "ru"
    "sk"
    "es"
    "ja"
    "cs"
-   "bg"},
+   "ar"
+   "bg"
+   "ko"},
  :packages ["metabase"],
```

Comment:
what causes this file to change?

Reply:
I actually have no idea, and hadn't noticed it. I'll revert this.